english/forest/shelter/pet-bear/pet-bear.md

```diff
@@ -2,4 +2,6 @@ You reach towards the bear's head with your right hand. This startles the
 bear. It scurries backwards out of the tent and runs away.
 
 
-[Get out of the shelter.](../play-dead/get-out/get-out.md)
```

Comment:
The only reason this is being deleted is because it does not have a carriage return. My guess is that this will not be accepted by @casey-collab.